libsurfman/src/xc.c

```diff
@@ -114,3 +96,8 @@ int xc_hvm_get_dirty_vram(int domid, uint64_t base_pfn, size_t n,
 {
   return xc_hvm_track_dirty_vram (xch, domid, base_pfn, n, db);
 }
+
+int xc_hvm_pin_memory_cacheattr(int domid, uint64_t pfn_start, uint64_t pfn_end, uint32_t type)
```

Comment:
Is this wrapper needed or could the call sites use `xc_domain_pin_memory_cacheattr` directly?

Reply:
Somewhat, to access the `static xc_interface *xch` variable which is initialized in `xc_init` and not exported.